src/styles/blocks/card.css

```diff
@@ -0,0 +1,74 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px 0 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-info {
+  text-align: start;
+  margin: 0 16px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--black-color-for-text);
+  margin-bottom: 5px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 11px;
+  color: var(--grey-color-for-text);
+  margin-bottom: 18px;
+}
+
+.card__price-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px ;
+}
+
+.card__price {
+  font-size: 12px;
+  color: var(--grey-color-for-text);
+}
+
+.card__amount {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--black-color-for-text);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  padding: 12px 0 ;
+  margin-bottom: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  background-color: var(--blue-BUY-button-color);
```

Comment:
hm.. if I wanna use same color for another button I need to create one node variable?